.gitignore

```diff
@@ -65,8 +65,5 @@ target/
 # MindMeld Models
 .generated/
 
-# MindMeld Data
-data/
```

Comment:
Why are we removing this?

Reply:
I'm not sure where this change came from. If I look at the history of .gitignore, I never had data/ there.

Reply:
No, let's remove it.